src/platform_impl/windows/window_state.rs

```diff
@@ -24,7 +23,7 @@ pub struct WindowState {
     pub saved_window: Option<SavedWindow>,
     pub dpi_factor: f64,
 
-    pub fullscreen: Option<MonitorHandle>,
+    pub fullscreen: Option<Fullscreen>,
```

Comment:
Placing this comment here, since github won't let me do it in the actual flags struct: exclusive fullscreen windows should set the `ALWAYS_ON_TOP` flag. The best way to do that is probably to add a `FULLSCREEN_OR_MASK` field, include `ALWAYS_ON_TOP` in it, and apply the mask in `fn mask`.

Reply:
I tried doing this earlier, but it didn't seem to make any difference in behaviour whatsoever. It made me think that maybe the window style for it is not getting applied correctly.. I didn't look into it any further.

Reply:
Really? This patch seems to work on my machine:

```patch
diff --git a/src/platform_impl/windows/window_state.rs b/src/platform_impl/windows/window_state.rs
index 67317c89..9aabd91d 100644
--- a/src/platform_impl/windows/window_state.rs
+++ b/src/platform_impl/windows/window_state.rs
@@ -75,6 +75,7 @@ bitflags! {
             WindowFlags::RESIZABLE.bits |
             WindowFlags::MAXIMIZED.bits
         );
+        const FULLSCREEN_OR_MASK  = WindowFlags::ALWAYS_ON_TOP.bits;
         const NO_DECORATIONS_AND_MASK = !WindowFlags::RESIZABLE.bits;
         const INVISIBLE_AND_MASK = !WindowFlags::MAXIMIZED.bits;
     }
@@ -210,6 +211,7 @@ impl WindowFlags {
     fn mask(mut self) -> WindowFlags {
         if self.contains(WindowFlags::MARKER_FULLSCREEN) {
             self &= WindowFlags::FULLSCREEN_AND_MASK;
+            self |= WindowFlags::FULLSCREEN_OR_MASK;
         }
         if !self.contains(WindowFlags::VISIBLE) {
             self &= WindowFlags::INVISIBLE_AND_MASK;
```